internal/operators/openshiftlogging/openshift_logging_manifests.go

```diff
@@ -0,0 +1,99 @@
+package openshiftlogging
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
+
+// Manifests returns manifests needed to deploy OpenShift Logging Operator
+func Manifests(cluster *common.Cluster) (map[string][]byte, []byte, error) {
+	loggingNamespace, err := getNamespace()
+	if err != nil {
+		return nil, nil, err
+	}
+	loggingOperatorGroup, err := getOperatorGroup()
+	if err != nil {
+		return nil, nil, err
+	}
+	loggingSubscription, err := getSubscription()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	openshiftManifests := make(map[string][]byte)
+	openshiftManifests["50_openshift-logging_ns.yaml"] = loggingNamespace
+	openshiftManifests["50_openshift-logging_operator_group.yaml"] = loggingOperatorGroup
+	openshiftManifests["50_openshift-logging_subscription.yaml"] = loggingSubscription
+
+	// No custom manifest needed for OpenShift Logging operator itself
+	return openshiftManifests, nil, nil
+}
+
+func getNamespace() ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE": Namespace,
+	}
+	return executeTemplate(data, "OpenShiftLoggingNamespace", OpenShiftLoggingNamespace)
+}
+
+func getOperatorGroup() ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE": Namespace,
+	}
+	return executeTemplate(data, "OpenShiftLoggingOperatorGroup", OpenShiftLoggingOperatorGroup)
+}
+
+func getSubscription() ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":         Namespace,
+		"OPERATOR_SUBSCRIPTION_NAME": SubscriptionName,
+		"OPERATOR_SOURCE":            Source,
+		"OPERATOR_SOURCE_NAMESPACE":  SourceNamespace,
+		"OPERATOR_CHANNEL":           Channel,
+	}
+	return executeTemplate(data, "OpenShiftLoggingSubscription", OpenShiftLoggingSubscription)
+}
+
+func executeTemplate(data map[string]string, contentName, content string) ([]byte, error) {
+	tmpl, err := template.New(contentName).Parse(content)
+	if err != nil {
+		return nil, err
+	}
+	buf := &bytes.Buffer{}
+	err = tmpl.Execute(buf, data)
+	if err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+const OpenShiftLoggingNamespace = `apiVersion: v1
```

Comment:
We have new mechanism now for templating the manifests in operators, please see other new operators examples